test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12639,7 +12639,26 @@ def isUp(self):
         """
         Check for PBS mom up
         """
-        return super(MoM, self)._isUp(self)
+        # Poll for few seconds so we don't miss any signals sent to
+        # mom
+        for _ in range(5):
+            rv = super(MoM, self)._isUp(self)
+            if rv:
+                try:
+                    nodes = self.server.status(NODE, id=self.shortname)
+                    if nodes:
+                        n_stat = self.server.expect(NODE,
+                                                    {'state': 'free'},
+                                                    id=self.shortname,
+                                                    max_attempts=5)
+                        rv = True
+                except PbsStatusError:
+                    pass
```

Comment:
I'm not sure why you pass on the PbsStatusError.  You'll fall right out and return True (since _isUp() returned that).  The server could be coming up and that's why the status() failed.  This will return a false positive.

Reply:
I passed on PbsStatusError to handle the case where there are no mom nodes. like some tests do a qmgr -c ‘d n @default’ and then restart mom. Mom is up, server is up, but no nodes.
In cases where the status errored due to server coming up, I guess we could anyways catch it in the 20s window polling.

Reply:
Ahh got it.  Maybe add a comment on why you pass?